implementation/jobberknoll/core/shared/uuid.ts

```diff
@@ -0,0 +1,19 @@
+import { generate, validate } from "@std/uuid/unstable-v7";
```

Comment:
sus

Reply:
Since it's a part of the standard library, there are strict stabilization guidelines.

As an alternative, I can import something like `npm:uuid`, but I'm guessing that an unstable lib from runtime authors is better than a stable but entirely community-supported lib.

Reply:
![image](https://github.com/user-attachments/assets/513c145a-8449-48ef-8650-e9dbc89db6a2)

Reply:
Sure, will change 👌

Reply:
If you say that you trust this "unstable" more than from the community, then fine, it can stay